pkg/cmd/bootstrap.go

```diff
@@ -3,6 +3,7 @@ package cmd
 import (
 	"errors"
 	"fmt"
+	"github.com/redhat-developer/kam/pkg/pipelines/imagerepo"
```

Comment:
This is not the common way to do imports in Go.

From  https://github.com/golang/go/wiki/CodeReviewComments#imports 

>  The standard library packages are always in the first group.

I'd move this line down a bit (`goimports -w` will fix this for you automatically).

Reply:
Odd, I'm using Goland.  Originally it was down with the other similar kam imports. Not sure how it got auto-added to the top there. 

Fixed.